### PR TITLE
Update to OC 1.0.0-rc2-14802

### PR DESCRIPTION
**Pull request by scleaver**
@domonkosgabor has requested this PR to align the Vue.js module with an OC release required for Kast 1.0.0-rc2-14802.

### CODE REVIEW

**Comment by scleaver** [comment]
Also, the FxCopAnalyzers has been added because we are using Lombiq.Analyzers

File: Lombiq.VueJs.csproj [hunk 1]
```diff
@@ -18,9 +18,9 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(OrchardCoreLibrariesRoot)' == '' AND '$(OrchardCoreModulesRoot)' == ''">
-    <PackageReference Include="OrchardCore.DisplayManagement" Version="1.0.0-rc2-13450" />
-    <PackageReference Include="OrchardCore.Module.Targets" Version="1.0.0-rc2-13450" />
-    <PackageReference Include="OrchardCore.ResourceManagement" Version="1.0.0-rc2-13450" />
+    <PackageReference Include="OrchardCore.DisplayManagement" Version="1.0.0-rc2-14802" />
+    <PackageReference Include="OrchardCore.Module.Targets" Version="1.0.0-rc2-14802" />
+    <PackageReference Include="OrchardCore.ResourceManagement" Version="1.0.0-rc2-14802" />
```
Comment (Piedone) [inline Lombiq.VueJs.csproj:23]: I suggest doing these changes in a fork. We keep the dev branch of our modules on the latest released Orchard version so updating to an ad-hoc preview version is not really suitable.

File: Lombiq.VueJs.csproj [hunk 2]
```diff
@@ -39,6 +39,12 @@
     <Folder Include="wwwroot\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.0">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+  </ItemGroup>
```
Comment (Piedone) [inline Lombiq.VueJs.csproj:47]: This is not necessary. If you add the Analyzers project [as documented](https://github.com/Lombiq/.NET-Analyzers#how-to-add-the-analyzers-to-you-project-repository) then this package and all the other analyzers will automatically be included in all projects of the given solution. If you use our [OSOCE](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) repo for developing these modules that happens as well.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.VueJs.csproj (hunk 1 above)
Comment (scleaver) [reply Lombiq.VueJs.csproj:23]: No problem... a fork is what I have been doing for quite a few months and happy to do it this way. @domonkosgabor asked me to submit a pull request so I did :)

File: Lombiq.VueJs.csproj (hunk 2 above)
Comment (scleaver) [reply Lombiq.VueJs.csproj:47]: Tha's strange huh. This was added automatically by your Analyzers project I think. Pretty sure I followed the documentation. I will remove - thanks for the advice.

**Pull request closed without merging**